playbooks/common/openshift-cluster/additional_config.yml

```diff
@@ -24,7 +24,3 @@
   - role: cockpit
     when: not openshift.common.is_atomic and ( deployment_type in ['atomic-enterprise','openshift-enterprise'] ) and
       (osm_use_cockpit | bool or osm_use_cockpit is undefined )
-  - role: flannel_register
```

Comment:
@detiber okay, then I'll remove the "Configure flannel" play.

In flannel_register role we are populating the networking template in the etcd db. This holds good even if we run on one master assuming all the masters are linked with same etcd api(cluster)